suite-native/discovery/src/utils.ts

```diff
@@ -1,18 +1,34 @@
 import { A } from '@mobily/ts-belt';
 
-import { Network, normalizeNetworkAccounts } from '@suite-common/wallet-config';
+import { AccountType, Network, normalizeNetworkAccounts } from '@suite-common/wallet-config';
 import { Account } from '@suite-common/wallet-types';
 
-export const getNetworksWithUnfinishedDiscovery = (
-    enabledNetworks: readonly Network[],
-    accounts: Account[],
-    accountsLimit: number,
-) =>
+export const getNetworksWithUnfinishedDiscovery = ({
+    enabledNetworks,
+    accounts,
+    accountsLimit,
+    availableCardanoDerivations,
+}: {
+    enabledNetworks: readonly Network[];
+    accounts: Account[];
+    accountsLimit: number;
+    availableCardanoDerivations?: AccountType[];
+}) =>
     enabledNetworks.filter(network =>
         // if there is no account for this network -> we should have at least one account even if added via Add Coin
         // or if there is at least one visible account of this type, we should have at least one hidden account so adding funds outside of this app is detected and account shown
-        // the whole network is considered undiscovered if this is found for either 'normal' account, or any of the other accountTypes
+        // the whole network is considered undiscovered if this is found for either 'normal' account, or any of supported accountTypes
         normalizeNetworkAccounts(network).some(({ accountType }) => {
+            // coinjoin and some cardano derivation might not be supported (cardano derivations are provided)
+            if (
+                'coinjoin' === accountType ||
+                (network.symbol === 'ada' &&
```

Comment:
Maybe rather `networkType` for consistency? Such pattern is used in `discoveryThunks.ts`.
Also, this could maybe break `tada`, which is the only other cardano-type network.

```suggestion
                (network.networkType !== 'cardano' &&
```